### ISSUE TITLE
Yu-Gi-Oh

### ISSUE THREAD

**Issue by PatrickvL** [Cxbx-Reloaded/game-compatibility#411]
### Description
According to @Strelok "Yu-Gi-Oh is looking very good with the pixel shader code"

```

```

### Status
**Previous behavior:** 
None reported

**Current behavior:**
Shows intro screens and menu 

### Screenshots

![](https://media.discordapp.net/attachments/293096509019258880/367215483168227328/Cxbx_2017-10-10_04-06-51-55.png)
![](https://media.discordapp.net/attachments/293096509019258880/367215695773171712/Cxbx_2017-10-10_04-07-25-44.png)
![](https://media.discordapp.net/attachments/293096509019258880/367215698088296448/Cxbx_2017-10-10_04-08-49-08.png)
![](https://media.discordapp.net/attachments/293096509019258880/367215703717314580/Cxbx_2017-10-10_04-07-34-27.png)
![Batman Dark Tomorrow menu?](https://media.discordapp.net/attachments/293096509019258880/367215713045315586/Cxbx_2017-10-10_04-34-37-32.png)
![](https://media.discordapp.net/attachments/293096509019258880/367215716581113856/Cxbx_2017-10-10_04-10-23-10.png)

### Additional Information
Tested with https://github.com/PatrickvL/Cxbx-Reloaded/commit/87dd49cde89be7611571c9c71498327ab97129d2
This branch is not yet merged into Cxbx-Reloaded master branch.

**Comment by literalmente-game** [comment]
I've sent Patrick the xbe dump, maybe it's better for him to edit the issue and add the dump himself than me adding in a comment?

**Comment by MarcoLoves360** [comment]
**Description**
this title used to go in-game before now it doesnt render anything

**Dumps**
[Xbe.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/2031454/Xbe.txt)
[KrnlDebug.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/2031460/KrnlDebug.txt)

**Status**

Previous behavior:
in-game

Current behavior:
shows nothing just black screen

**Additional Information**

build 72c9132f

**Comment by MarcoLoves360** [comment]
Description
its working again!!

Dumps
[Xbe.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/2049262/Xbe.txt)
[KrnlDebug.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/2049264/KrnlDebug.txt)

Status

Previous behavior:
black screen

Current behavior:
in-game looks very good, some models look very white. when you win a duel the three egyptian goods obelisk wont render the others look very good. sometimes crash at the start or end of a duel with a pop up window (see below) other times it just go black at any point.

Screenshots
![image](https://user-images.githubusercontent.com/30830822/40672258-2526b078-6366-11e8-8f50-b5bdd008be16.png)

![image](https://user-images.githubusercontent.com/30830822/40672177-ec44759c-6365-11e8-8026-556516252d8e.png)

![image](https://user-images.githubusercontent.com/30830822/40672434-b4d6e4a4-6366-11e8-92ff-f490c60c317e.png)

Additional Information

build 601ba35c

**Comment by Janrick17** [comment]
Description
I'm newbie here 

Dumps
What is dumps?

Status:
Black screen only

Current behavior:
After input this game and run it emulation shows nothing just black screen, can you help me how to working this game?

**Comment by xPhilipDK** [comment]
I'm unable to emulate the game. As soon as i start, the screen remains black and the emulator is not responding

**Comment by Janrick17** [comment]
OMG! That game it works from latest version, But as soon as started the issue of frame rate is very bad, While the duels begin turns very slow max fps is 9. What should I do? Upgrade my PC?

See attached snapshot frame rate is bad.

My PC specs:
i5-4440  (3.10 GHz)
GTX 650
8 GB RAM DDR4 3000MHz
500 HDD Storage

![yu gi oh-The dawn of destiny](https://user-images.githubusercontent.com/40024460/67759906-49f19000-fa7b-11e9-8d04-0ee44e94e1c5.JPG)

**Comment by hyperspeedgx** [comment]
Any settings needed to run this? Here it gives me Black Screen.

**Comment by ObiKKa** [comment]
Just an important reminder that after PatrickvL's palettized textures conversion, [PR#1791](https://github.com/Cxbx-Reloaded/Cxbx-Reloaded/pull/1791) (Nov 22, 2019), that led to a huge speedup boost for several (maybe many more than thought initially) titles, this game here now runs full speed.

- [Showcase after performance improvements](https://m.youtube.com/watch?v=0Uy9V2Mm43M) (literalmente{game}.  Published on 23 Nov 2019.

List of tested games:
- Auto Modellista
- Memorick - The Apprentice Knight
- Red Faction II
- Spikeout Battle Street
- Starsky And Hutch
- Total Overdose
- Yu-Gi-Oh - The Dawn of Destiny)

**Comment by marcosviniciusmv** [comment]
YuGiOh, in the compilation CI-f3d7ae6, I managed several duels and the FPS drops dramatically when it will choose a statue to win some card but still not crash. Well, I don't know if it helps, but I only managed to play
these duels specifically in that build. In any other build it crashes after the first duel and the emulation stops.

**Comment by marcosviniciusmv** [comment]
Can someone explain to me why the emulator can run games that require heavier graphics and cannot run YuGiOh which is "less demanding"?

**Comment by Margen67** [comment]
@marcosviniciusmv Off-topic discussion belongs on the [Discord server](https://discord.gg/26Xjx23).

**Comment by FerPozzetti** [comment]
As the good news bringer, I come here to state that the game runs almost perfect (at least the first duel; I haven´t played it more than that).

Version f1d09af (Jun  8 2020)

The only small detail that I saw is that certain textures render with a thin black bar. I was not using native res, so, you can probably blame it on that.
 
![4](https://user-images.githubusercontent.com/56910468/84731943-c71d3700-af70-11ea-8c16-a45423c74bfa.jpg)

![3](https://user-images.githubusercontent.com/56910468/84731941-c4224680-af70-11ea-865d-13145301be2d.jpg)


![2](https://user-images.githubusercontent.com/56910468/84731928-c08ebf80-af70-11ea-8e69-d50d32c43088.jpg)

![1lo](https://user-images.githubusercontent.com/56910468/84731921-bcfb3880-af70-11ea-93f7-0cc137e1c672.jpg)





[KrnlDebug.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/4784361/KrnlDebug.txt)
[Xbe.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/4784362/Xbe.txt)

**Comment by marcosviniciusmv** [comment]
> As the good news bringer, I come here to state that the game runs almost perfect (at least the first duel; I haven´t played it more than that).
> 
> Version f1d09af (Jun 8 2020)
> 
> The only small detail that I saw is that certain textures render with a thin black bar. I was not using native res, so, you can probably blame it on that.
> 
> ![4](https://user-images.githubusercontent.com/56910468/84731943-c71d3700-af70-11ea-8c16-a45423c74bfa.jpg)
> 
> ![3](https://user-images.githubusercontent.com/56910468/84731941-c4224680-af70-11ea-865d-13145301be2d.jpg)
> 
> ![2](https://user-images.githubusercontent.com/56910468/84731928-c08ebf80-af70-11ea-8e69-d50d32c43088.jpg)
> 
> ![1lo](https://user-images.githubusercontent.com/56910468/84731921-bcfb3880-af70-11ea-93f7-0cc137e1c672.jpg)
> 
> [KrnlDebug.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/4784361/KrnlDebug.txt)
> [Xbe.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/4784362/Xbe.txt)

No, the game is not running perfectly, there are freezes of animations after some time playing, and the triple duels always after winning the first opponent the emulation stops or freezes, on the statues screen also after winning or losing a duel the fps drops to 2-5. I mentioned that in the compilation CI-f3d7ae6 I managed to finish 3 duels without errors, but I don't know if the comment was of great help. But the game is not running perfectly.

**Comment by AvZ77** [comment]
Yea the frame rate drop to like 2 fps or so at choosing a statue after the duel but am I the only one having issue with sound?
Sound tracks plays only for like 3 or 4 seconds before it repeats, gets very annoying after a while.
Seems to run well on VS2017 but it crashes after a few minutes on VS2019.
These are the issues that I can see, otherwise it runs rather well.
Build CI-97bf1d9

**Comment by marcosviniciusmv** [comment]
> Yea the frame rate drop to like 2 fps or so at choosing a statue after the duel but am I the only one having issue with sound?
> Sound tracks plays only for like 3 or 4 seconds before it repeats, gets very annoying after a while.
> Seems to run well on VS2017 but it crashes after a few minutes on VS2019.
> These are the issues that I can see, otherwise it runs rather well.
> Build CI-97bf1d9


I can't get past the first duel. Always when the error duel ends. Is it working normally? I tested the build you mentioned but the error persists.

**Comment by AvZ77** [comment]
> 
> 
> > Yea the frame rate drop to like 2 fps or so at choosing a statue after the duel but am I the only one having issue with sound?
> > Sound tracks plays only for like 3 or 4 seconds before it repeats, gets very annoying after a while.
> > Seems to run well on VS2017 but it crashes after a few minutes on VS2019.
> > These are the issues that I can see, otherwise it runs rather well.
> > Build CI-97bf1d9
> 
> I can't get past the first duel. Always when the error duel ends. Is it working normally? I tested the build you mentioned but the error persists.

No I tried again several times and indeed it does crash now even for me, it was a lucky run the first time getting as far as completing one duel I guess.
